tables/tests/test_utils.py

```diff
@@ -4,7 +4,7 @@
 try:
     from unittest.mock import patch
 except ImportError:
-    from mock import patch
+    from unittest.mock import patch
```

Comment:
Forgot this?
Please just replace:
```
try:
    from unittest.mock import patch
except ImportError:
    from mock import patch
```
with
```
from unittest.mock import patch
```

Reply:
Right, thanks. There were two other cases of now-useless `ImportError` hacks. Fixed all of them.